integration_tests/schemaops/utestschema/helper.go

```diff
@@ -50,3 +50,52 @@ func GetDeviceWithOrderedMap(t *testing.T) *Device {
 		},
 	}
 }
+
+func GetOrderedMap2(t *testing.T) *Ctestschema_OrderedLists_OrderedList_OrderedMap {
```

Comment:
godoc  
nit: i don't like "2" suffix, it's unclear what makes this func different from other func without looking at the code

Reply:
Thanks for the feedback, I've added a godoc. I think in this case it's ok to leave it like this rather than `GetOrderedMapWeeWoo` as the readers are mostly ygot maintainers and the code is fairly easy to understand. The "2" as a indicator for a different ordered map I think is sufficient. Let me know if you have other thoughts on this.